controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -458,6 +459,16 @@ func (r *Reconciler) checkIsTargetValid(ctx *pkgctx.VirtualMachinePublishRequest
 			return err
 		}
 		contentLibraryType = contentLibraryV1A2.Spec.Type
+
+		if contentLibraryType == imgregv1.LibraryTypeInventory && contentLibraryV1A2.Labels != nil {
```

Comment:
would metav1.HasLabel() be helpful here?

Reply:
sure. it won't provide any functional benefit, but would prettify things a tad.